Frontend/src/app/search-response/search-response.component.html

```diff
@@ -0,0 +1,46 @@
+
+
+<div class="container">
+  <table class="table">
```

Comment:
![image](https://user-images.githubusercontent.com/75213424/154852028-f3c12d12-a33c-45f3-afc3-9a6a190fa546.png)
you should add mt-1 for table. It looks better

Reply:
Subjectively

Reply:
ok. just advice